### DISCUSSION TITLE
Prowler JSON output is great

### DISCUSSION THREAD

**Discussion by rascal999** [prowler-cloud/prowler#2464] [General]
I just wanted to thank the team who developed this tool, and a special shout out to the people who worked on the JSON schema. "FindingUniqueId" was a stroke of genius, because it makes it very easy to check for duplicates. I wish more tools had JSON output like Prowler.

Thanks again!

**Comment by toniblyx** [comment]
Thanks for you kind words @rascal999, writing software we make good and bad decisions every day, glad to see that the json format was one of the good ones :) Thanks again.
